devel/external/AEGIS/README.md

```diff
@@ -0,0 +1,54 @@
+* Co-added images : The science images in multiple bands should be added. The variance maps (1/weight map) of multiple bands should be added. Objects are detected on these Co-added images using SExtractor in dual image mode.  
+* Tiny Tim Images: Images of PSF at different locations in the focal plane for different focus offsets. The PSF field files for the multiple filters should be in  separate folders.
+* File with regions for manual masks. These are regions that upon visual inspection were found to contain artifacts e.g: reflection ghosts, and hence need to be masked out. The text file contains segment ID, filter, x and y coordinates of the points on a quadrilateral that marks the region to be masked, file name of science image with region to be masked. Note: A separate script identifies and masks regions near saturated stars; hence stellar masks need not be included in manual masks.
+* Noise map: Fits file with the noise covariance matrix for each band. Empty regions in the science image were visually identified from which the covariance matrix of the noise was computed.(See ipython notebook check_empty_regions.ipynb)
+* Other catalogs (OPTIONAL): The script also has provision to add information from other catalogs to the main catalog, e.g:redshift information from another catalog.
+
+### Input Parameters for script:
+* zero_point_mag: Zero point magnitude for each filter.
+* diff_spike_params: These parameters are used to compute the size of the diffraction spikes. A polygonal mask is drawn around saturated stars to mask these spikes (see Fig1: in document).  Parameters are [slope(pixels/ADU), intercept(pixels),width(pixels),angle(degrees)]. Slope and intercept relate the FLUX_AUTO of the star to the length the spike (Obtained from a linear fit to FLUX_AUTO Vs length of the spike measured manually for 10 saturated stars). The width of the spikes is set with width. Angle gives the angle by which the polygon has to be rotated. The parameters may be different for different bands. 
+* star_galaxy_params: Parameters used to separate galaxies and stars in MU_MAX Vs MAG_AUTO plot(x_div, y_div, slope). x_div gives the maximum magnitude, below which the object is saturated. y_div is the value of surface brightness per pixel for a saturated star. The slope of the line separating stars and galaxies is given by slope (See Fig 2 in document). It is recommended to run the first script get_objects.py, with mock sat_galaxy_params values, on a small region and then compute the separation parameters from the objects measured. The parameters may be different for different bands.
+* gain: Detector gain in e/ADU 
+
+### Scripts: 
+The scripts are run entirely in python. Make sure the following modules are loaded:
+galsim, numpy, astropy, asciidata, subprocess, os, scipy.
+The detection and measurement is performed with [SExtractor](http://www.astromatic.net/software/sextractor)
```

Comment:
Any restrictions on what sextractor version is needed in order for this to work?